src/components/clusterprovider/minikube/minikube.ts

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+import * as vscode from 'vscode';
+import { shell, ShellResult } from '../../../shell';
+
+export async function startMinikube() {
+    shell.exec('minikube start').then((result: ShellResult) => {
+        if (result.code == 0) {
+            vscode.window.showInformationMessage('Cluster started.');
+        } else {
+            vscode.window.showErrorMessage(`Failed to start cluster ${result.stderr}`);
+        }
+    }).catch((err) => {
+        vscode.window.showErrorMessage(`Failed to start cluster: ${err}`);
+    });
+}
+
+export async function stopMinikube() {
+    shell.exec('minikube stop').then((result: ShellResult) => {
+        if (result.code == 0) {
+            vscode.window.showInformationMessage('Cluster stopped.');
+        } else {
+            vscode.window.showErrorMessage(`Error stopping cluster ${result.stderr}`)
```

Comment:
I know this is persnickety but the linter is still flagging a missing semicolon here.

Reply:
Do you have the VS Code `TS Lint` extension installed?  It will give you red wigglies for things like this.

Reply:
done. thanks for the tip on the linter, I have it installed now :)